pdns/dnsbackend.hh

```diff
@@ -50,22 +50,22 @@ struct SOAData;
 
 struct DomainInfo
 {
-  DomainInfo() : last_check(0), backend(nullptr), id(0), notified_serial(0), receivedNotify(false), serial(0), kind(DomainInfo::Native) {}
+  DomainInfo() : kind(DomainInfo::Native) {}
 
   DNSName zone;
   DNSName catalog;
-  time_t last_check;
+  time_t last_check{};
   string options;
   string account;
-  vector<ComboAddress> masters; 
-  DNSBackend *backend;
+  vector<ComboAddress> masters;
+  DNSBackend *backend{};
 
-  uint32_t id;
-  uint32_t notified_serial;
+  uint32_t id{};
+  uint32_t notified_serial{};
 
-  bool receivedNotify;
+  bool receivedNotify{};
 
-  uint32_t serial;
+  uint32_t serial{};
```

Comment:
It's a matter of taste so we probably need to discuss this with the team, but I really prefer using explicit values (`0`, `nullptr`, `0`, `0`, `false`, `0`).